core/unit_test/incremental/Test12b_TeamScratch.hpp

```diff
@@ -107,11 +107,13 @@ struct TeamScratch {
 
 TEST(TEST_CATEGORY, IncrTest_12b_TeamScratch) {
   TeamScratch<TEST_EXECSPACE> test;
-  test.run(1, 4, 4);
-  test.run(4, 7, 10);
 #ifdef KOKKOS_ENABLE_OPENMPTARGET
-  test.run(14, 277, 1);
+  test.run(1, 32, 4);
+  test.run(4, 64, 10);
+  test.run(14, 128, 20);
```

Comment:
and this?

Reply:
Same as above, the team_size needs to be a multiple of 32 for Release builds.